phpstan-baseline.neon

```diff
@@ -0,0 +1,31 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Variable \\$value on left side of \\?\\? always exists and is always null\\.$#"
+			count: 1
+			path: src/Eloquent/Model.php
+
+		-
+			message: "#^Variable \\$value on left side of \\?\\? always exists and is not nullable\\.$#"
+			count: 1
+			path: src/Eloquent/Model.php
+
```

Comment:
```suggestion

```
You can remove these two errors after applying this [change](https://github.com/mongodb/laravel-mongodb/pull/2664/files#r1403260408).